pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -706,26 +728,74 @@ private void recordIndexingError(String indexType) {
 
   private void aggregateMetrics(GenericRow row, int docId) {
     for (MetricFieldSpec metricFieldSpec : _physicalMetricFieldSpecs) {
-      String column = metricFieldSpec.getName();
+      String column = _ingestionAggregator.getMetricName(metricFieldSpec.getName());
```

Comment:
Suggest renaming this to `sourceColumn`, and keep the current `column`

Reply:
The line below is the only place it's used so I just referenced it directly there.